dhis-2/pom.xml

```diff
@@ -463,6 +466,9 @@
 
         <profile>
             <id>dev</id>
+            <properties>
+                <skipTests>true</skipTests>
```

Comment:
what is the dev mode actually for? not sure if people expect the tests to run

Reply:
Exactly.
The tests are skipped.
This profile is used to build the image for the api tests

Reply:
I think the profile can be removed if that's the only reason it's there. We can always do -DskipTests and not use a profile. I would agree that `dev` might implicitly mean that unit tests are included. But that's not necessary for API tests.